source/deploy/mattermost-changelog.md

```diff
@@ -15,6 +15,74 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
 :depth: 2
 ```
 
+## Release v9.6 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: March 15, 2024**
+
+### Compatibility
+ - Updated minimum required Edge and Chrome versions to 120+.
+
+```{Important}
+If you upgrade from a release earlier than v9.5, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Pre-packaged Calls version [v0.24.0](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.24.0).
+ - Pre-packaged GitLab plugin version [v1.8.0](https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.8.0).
+ - Added the [Outgoing OAuth Connections](https://mattermost.com/pl/outgoing-oauth-connections) integration type.
+ - Re-designed the **System Console > User Management** screen, and added the ability to batch export users in CSV format.
```

Comment:
Should we note that part of this only works on Postgres? 

CC @devinbinnie @harshilsharma63 in case I'm misremembering

Reply:
users can still batch export report on MySQL, just not see the live results in UI IIRC